spec/integration/api/comments_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "spec_helper"
+
+describe Api::V0::PostsController do
+  let(:auth) { FactoryGirl.create(:auth_with_read_and_write) }
+  let!(:access_token) { auth.create_access_token.to_s }
+
+  before do
+    @status = auth.user.post(:status_message, text: "This is a status message", public: true, to: "all")
+  end
+
+  describe "#create" do
+    context "valid post ID" do
+      it "succeeds" do
+        post api_v0_post_comments_path(post_id: @status.id), text: "This is a comment", access_token: access_token
+        expect(JSON.parse(response.body)["text"]).to eq("This is a comment")
+      end
+    end
+
+    context "comment too long" do
+      before do
+        post api_v0_post_comments_path(post_id: @status.id), text: "This is a long comment" * 99999, access_token: access_token
```

Comment:
- Separate every 3 digits in the integer portion of a number with underscores(_).
- Line is too long. [127/120]